specs/lang_switcher.spec.ts

```diff
@@ -34,8 +34,8 @@ test('switching', async () => {
 
   // page path
   expect(await getData(page, '#home-use-async-data')).toMatchObject({
-    aboutPath: '/about',
-    aboutTranslation: 'About us'
+    aboutPath: '/fr/about',
+    aboutTranslation: 'À propos'
```

Comment:
oh, nuxt i18n has still bug.
Since your PR is an update of the deps module, we will accept it, and I’ll open the issue.

Reply:
I've just opened #2283